spec/factory_logger_spec.rb

```diff
@@ -1,12 +1,97 @@
 require 'spec_helper'
-
 RSpec.describe FactoryLog, type: :model do
-  describe 'test' do
-    it { expect(1 + 1).to eq 2 }
+  describe '#log_set' do
+    let!(:factoryAs) { create_list(:factory_log, 10) }
+    let!(:factoryBs) { create_list(:factory_log, 10, :factoryB) }
+
+    it 'return FactoyAs' do
+      expect(FactoryLog.log_set("factoryA")).to eq factoryAs
+    end
+  end
+
+  describe '#log_set_with_trait' do
+    let!(:factoryAs) { create_list(:factory_log, 10) }
+    let!(:factoryAs_with_trait) { create_list(:factory_log, 10, :have_trait) }
+    let!(:sample) { factoryAs_with_trait.first }
+
+    it 'return objects have trait' do
+      expect(FactoryLog.log_set_with_trait(sample.name + sample.traits)).to eq factoryAs_with_trait
+    end
+  end
+
+  describe '#candidates' do
+    let!(:factoryAs) { create(:factory_log) }
+    let!(:factoryAs_with_trait) { create(:factory_log, :have_trait) }
+    let!(:factoryBs) { create(:factory_log, :factoryB) }
+
+    it 'return all candidates' do
+      expect(FactoryLog.candidates()).to eq [
+                                              (factoryAs.name + factoryAs.traits),
```

Comment:
Line is too long. [82/80]<br>Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.